src/models/config/config.ts

```diff
@@ -18,6 +19,12 @@ export type ConfigLike = {
   repository: string;
 };
 
+export const ConfigSchema = z.object({
```

Comment:
issue(non-blocking): this is not a type nor a class; the name should be camel-case
```suggestion
export const configSchema = z.object({
```

Reply:
question(blocking): what will happen if one of the properties is missing in the configuration file? If it will fail - we're good. If it will pass, but values are empty/undefined - we must make these parameters required as specified in the documentation.

Reply:
Followed the official documentation [example](https://deno.land/x/zod@v3.20.2#basic-usage)

Reply:
It shows basic usage, we need to make sure that all values are presented, because name, email, and repository are required to be filled-out when reading from the config file. There's a method to make parameters required, if this method is optional then we don't need to specify it, otherwise we do. Would you check it out please?

Reply:
> issue(non-blocking): this is not a type nor a class; the name should be camel-case
> I followed the official documentation [example](https://deno.land/x/zod@v3.20.2#basic-usage)

Reply:
> It shows basic usage, we need to make sure that all values are presented, because name, email, and repository are required to be filled-out when reading from the config file. There's a method to make parameters required, if this method is optional then we don't need to specify it, otherwise we do. Would you check it out please?

I'll check it out now

Reply:
There's a bug related to #34 and will be handled separately.